daprdocs/content/en/operations/support/support-release-policy.md

```diff
@@ -57,15 +59,30 @@ General guidance on upgrading can be found for [self hosted mode]({{<ref self-ho
 |                          |                 1.1.2 |                    1.2.2 |
 |                          |                 1.2.2 |                    1.3.1 |
 |                          |                 1.3.1 |                    1.4.1 |
+|                          |                 1.4.1 |                    1.4.2 |
+|                          |                 1.4.2 |                    1.4.3 |
 | 1.1.0 to 1.1.2           |                   N/A |                    1.2.2 |
 |                          |                 1.2.2 |                    1.3.1 |
 |                          |                 1.3.1 |                    1.4.1 |
+|                          |                 1.4.1 |                    1.4.2 |
+|                          |                 1.4.2 |                    1.4.3 |
 | 1.2.0 to 1.2.2           |                   N/A |                    1.3.1 |
 |                          |                 1.3.1 |                    1.4.1 |
+|                          |                 1.4.1 |                    1.4.2 |
+|                          |                 1.4.2 |                    1.4.3 |
 | 1.3.0                    |                   N/A |                    1.3.1 |
 |                          |                 1.3.1 |                    1.4.1 |
+|                          |                 1.4.1 |                    1.4.2 |
+|                          |                 1.4.2 |                    1.4.3 |
 | 1.3.1                    |                   N/A |                    1.4.1 |
+|                          |                 1.4.1 |                    1.4.2 |
+|                          |                 1.4.2 |                    1.4.3 |
 | 1.4.0                    |                   N/A |                    1.4.1 |
+|                          |                 1.4.1 |                    1.4.2 |
```

Comment:
No need for all of these. See how v1.2.0-v1.2.2 are treated above (the only exception is v1.3.1 since that is actually needed to upgrade to v1.4)